service/templates/settings.html

```diff
@@ -86,41 +86,8 @@ <h3 style="padding-left:15px;"><i class="fa fa-globe"></i> User Settings</h3>
               </div>
             </div>
             <div class="form-group">
-              <label class="col-sm-4 control-label">Slack Username</label>
```

Comment:
Yes, but if we're just looking at UI it makes sense to have all the notification-related settings on a single page. Putting all of that on the user settings page would make it too crowded, so all of it is on a new page

Reply:
What i'm imagining in my head is that you leave these settings on the user settings page, and have the subscriptions list on another page. This way we avoid having too much on one page, but leave the settings in the intuitive location. (and i call it intuitive because i imagine the new page you've created to be a notifications listing page, not a notifications settings page).

Reply:
Yea, I agree that this is kind of weird. But I'd argue it's not about everything notification related in one spot, it's about presenting a page to edit and delete existing subscriptions. The settings on the user page dictate 1. where to deliver slack messages, and 2. how to deliver the implicit subscriptions.

Also consider this, we've already announced/told people to go to their user settings page to set things up. By moving things, we're going to confuse people.

Reply:
The thing is having the implicit + explicit subscriptions on one page gives the user a complete view of all the notifications they'll get. Splitting it up will be more confusing and could make it unclear that the 2 are the extent of notifications.

Reply:
filtered those out on the client